### ISSUE TITLE
Cooler Master MH630 equalizer makes the audio worse

### ISSUE THREAD

**Issue by 0N1rick** [jaakkopasanen/AutoEq#746]
**Describe the bug**
I downloaded the EquzlierAPO GraphicEq file for Cooler Master Mh630 headphones and correctly imported in the EqAPO config.
The audio is equalized but the result is worse than the original sound signature: the sound is dull, highs seems dampened...
I was looking at the graph on autoeq.app and the equalizer curve seems different from the parameters in the GraphicEq file, for example I see a boost in frequencies between 20 and 70 Hz in the graph but the same range has negative values in the APO file.
Not sure the problem is this, I am new to equalizing.

On the other hand, it works perfectly with my Beyerdynamic DT 700 Pro X, so the problem does not lie in EqualizerAPO itself or my config.

**To Reproduce**
Use EqualizerAPO GraphicEq file for Cooler Master MH630 headphones. Tested with both harman over-ear 2018 and HMS II.3 Harman over-ear 2018 targets.

**Expected behavior**
Final sound should be "better" (in regard to Harman profile) than the original.

**Files**
[Cooler Master MH630 GraphicEq.txt](https://github.com/jaakkopasanen/AutoEq/files/13545221/Cooler.Master.MH630.GraphicEq.txt)

**System:**
Windows 11 23H2
EquzlierAPO 1.3

**Comment by jaakkopasanen** [comment]
Have you tried other equalizer types, such as parametric eq?

**Comment by 0N1rick** [comment]
Nope, I tried it now. The result is the same.
But I figured out the issue! It was... me.

I got fooled by hedphone's original signature that has too many highs. That led me to think that your EQ made it worse, muffled.
Today I got a little more time to experiment, so I took my Beyerdynamic DT 700 Pro X and used them to compare.
I tested different scenarios (music, talk and game) and that helped me to understand that I was just wrong.

I am sorry to have wasted your time.
Thank you again for the awesome work. It's really marvelous! <3
